modules/react/text-input/stories/TextInput.stories.mdx

```diff
@@ -81,6 +82,22 @@ Labels for required fields are suffixed by a red asterisk.
 
 <ExampleCodeBlock code={Required} />
 
+### Icons
+
+`IconGroup` is available to add icons to the `TextInput`. Internally, a container `div` element is
```

Comment:
Did you mean `InputGroup` ?

Reply:
No. An `InputGroup` adds icons "inside" `TextInput` component

Reply:
where is icongroup as a component?

Reply:
Oh. I thought you meant `TextInput` should be `InputGroup`. You can use the suggestion feature to help narrow it down.
```suggestion
`InputGroup` is available to add icons to the `TextInput`. Internally, a container `div` element is
```